### ISSUE TITLE
[deleted]

### ISSUE THREAD

**Issue by mefferr** [vinceliuice/Orchis-theme#263]
[deleted]

**Comment by xan-wave** [comment]
nevermind, i fixed it by redownloading the orchis git and running the ./install.sh -u command

**Comment by xan-wave** [comment]
Also, do you know why i can't delete the WhiteSur-gtk-theme folder?

**Comment by vinceliuice** [comment]
You need download the WhiteSur-gtk-theme git and run `./install.sh -u` if you installed with sudo then you need run `sudo ./install.sh -u` also if you installed gdm theme then you need run `sudo ./tweaks.sh -g -r`

**Comment by 0x-j** [comment]
> You need download the WhiteSur-gtk-theme git and run `./install.sh -u` if you installed with sudo then you need run `sudo ./install.sh -u` also if you installed gdm theme then you need run `sudo ./tweaks.sh -g -r`

I'm having a similar issue. I originally had a light theme on the Files, Settings and Software apps, but I did some tweaking and removed the light themes from the .themes folder and it set them all to dark again, but with the original blue colors. I installed the Orchis Red Dark theme and it looks amazing on the top bar and top right widgets, but the red coloring won't take on those other apps. Are they running a different version of GTK? I'm still learning how to do all of this. Any help or advice will be much appreciated. Thank you.
